FC-0068 docs: reference and how-tos reviewed by edunext 20 (#771)
openedx/docs.openedx.org: source/educators/references/communication/example_bulk_emails.rst

omitting the discussion highlights if they are not appropriate for your course
(the example discussion highlights are from a course about urban solutions to
the global water crisis). Search for the values enclosed by {braces} and

> replace them with information for your course. In addition, check you are

Reviewer: ```suggestion
replace them with information for your course. In addition, check that you are
```

than one message while the course is running. You can use this template as a
starting point for your own messages; search for the values enclosed by
{braces} and replace them with information for your course. In

> addition, check you are using :ref:`keywords <Use Keywords in Messages>`

Reviewer: ```suggestion
addition, check that you are using :ref:`keywords <Use Keywords in Messages>`
```

  graders.

Be sure to search for the values enclosed by {braces} and replace them with

> information for your course. In addition, check you are using

Reviewer: ```suggestion
information for your course. In addition, check that you are using
```

  Thank you for making %%COURSE_DISPLAY_NAME%% so much fun these last few
  months! We had a great time getting to know you through the course
  discussions. We appreciate the effort you put into this course, and we
  hope you enjoyed learning {subject}. With or without a
  certificate, you should be proud of your accomplishments.

  * Please take a few minutes to answer the exit survey, now available on the
    Course page. We will use your responses to improve the course in the
    future.

  * If you qualify for a certificate (overall score {number}% or higher), the
    dashboard will include a link to your certificate in the near future.
    While you may see the link in a few days, it can take up to two weeks for

>     platform to generate all of the course certificates.

Reviewer: ```suggestion
    all of the course certificates to be generated.
```
